File: NetBeans IDE/Evaluacion 2/EVA2_4_NESTED_IF_2/C++/main.cpp
#include <iostream>
#include <sstream>
#include <string>
#include <cstdlib>
#include <cmath>

using namespace std;

// Headers
string toString (double);
int toInt (string);
double toDouble (string);

int main() {
    cout << "Introduce tu calificacion:" << endl;
    int iCali;
    
    cin >> iCali;
    if (iCali == 100) {
        cout << "A" << endl;
    } else {
        if (iCali >= 90 && iCali < 100) {
            cout << "B" << endl;
        } else {
            if (iCali >= 80 && iCali < 90) {
                cout << "C" << endl;
            } else {
                if (iCali >= 70 && iCali < 80) {
                    cout << "D" << endl;
                } else {
                    if (iCali >= 0 && iCali < 70) {
                        cout << "F" << endl;
                    } else {
                        cout << "Fuera de rango" << endl;
                    }
                }
            }
        }
    }
    return 0;
}

// The following implements type conversion functions.
string toString (double value) { //int also
    stringstream temp;
    temp << value;
    return temp.str();
}

int toInt (string text) {
    return atoi(text.c_str());
}

double toDouble (string text) {
    return atof(text.c_str());
}

File: NetBeans IDE/Evaluacion 2/EVA2_4_NESTED_IF_2/Python/main.py
print("Introduce tu calificacion:")
iCali = int(input())
if iCali == 100:
    print("A")
else:
    if iCali >= 90 and iCali < 100:
        print("B")
    else:
        if iCali >= 80 and iCali < 90:
            print("C")
        else:
            if iCali >= 70 and iCali < 80:
                print("D")
            else:
                if iCali >= 0 and iCali < 70:
                    print("F")
                else:
                    print("Fuera de rango")
